src/lib/MUIRating.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { IconContainerProps, Rating, RatingProps } from '@material-ui/lab'
+import { Box, Typography } from '@material-ui/core';
+import { attachField, IFieldProps } from 'react-forms'
+import { FormikValues } from 'formik';
+import _ from 'lodash';
+
+export interface IMUIRating extends RatingProps {
```

Comment:
Export it as MuiRatingProps. import RatingProps from material-ui with some other name